examples/03-advanced/06-stress_gradient_path.py

```diff
@@ -0,0 +1,144 @@
+"""
+.. _stress_gradient_path:
+
+Stress gradient normal to a defined node.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+This example shows how to plot a stress gradient normal to a selected node.
+As the example is based on creating a path along the normal, the selected node
+must be on the surface of the geometry.
+A path is created of a defined length.
+
+"""
+
+###############################################################################
+# First, import the DPF-Core module as ``dpf`` and import the
+# included examples file and ``DpfPlotter``
+#
+import matplotlib.pyplot as plt
+from ansys.dpf import core as dpf
+from ansys.dpf.core import operators as ops
+from ansys.dpf.core.plotter import DpfPlotter
+from ansys.dpf.core import examples
+
+###############################################################################
+# Next, open an example and print out the ``model`` object.  The
+# :class:`Model <ansys.dpf.core.model.Model> class helps to organize access
+# methods for the result by keeping track of the operators and data sources
+# used by the result file.
+#
+# Printing the model displays:
+#
+# - Analysis type
+# - Available results
+# - Size of the mesh
+# - Number of results
+# - Unit
+#
+model = dpf.Model(examples.hemisphere)
+print(model)
+###############################################################################
+# Define the `node_id` normal to which a stress gradient should be plotted.
+#
+node_id = 1928
+###############################################################################
+# The following command prints the mesh unit
+#
+unit = model.metadata.meshed_region.unit
+print("Unit: %s" % unit)
+###############################################################################
+# `depth` defines the path length / depth to which the path will penetrate.
+# While defining `depth` make sure you use the correct mesh unit.
+# `delta` defines distance between consecutive points on the path.
+depth = 10  # in mm
+delta = 0.1  # in mm
+###############################################################################
+# Get the meshed region
+#
+mesh = model.metadata.meshed_region
+###############################################################################
+# Get Equivalent stress fields container.
+#
+stress_fc = model.results.stress().eqv().outputs.fields_container()
+###############################################################################
+# Define Nodal scoping.
+# Make sure to define ``"Nodal"`` as the requested location, important for the
+# `normals` operator.
+#
+nodal_scoping = dpf.Scoping(location="Nodal")
```

Comment:
It might be hard to remember the right locations (and their case), this can help:
```suggestion
nodal_scoping = dpf.Scoping(location=dof.locations.nodal)
```